Automattic/woocommerce-payments
# Transactions - Use Pill component to display risk levels in transactions list

Fixes #6038 

#### Changes proposed in this Pull Request

Risk level column within `Payments > Transactions` to be displayed using the Pill component, instead of plain text with color. This is to make it consistent with UI guidelines of WC and WC Pay. 

#### Screenshots

##### Before 
![image](https://user-images.githubusercontent.com/4162931/231967091-2eaddb53-b9be-4ba1-9984-ee7065d0abd4.png)

##### After
![image](https://user-images.githubusercontent.com/4162931/231967165-25eb730c-8f1f-4b3e-979e-40a13d3620ad.png)


#### Testing instructions

1. Checkout this branch and do a few transactions using normal and Risky (fraud testing ) test cards
2. Browse to `Payments > Transactions`
3. Unhide the Risk Level column, if its hidden.
4. The Risk level should be visible within a Pill component. 
5. Test the same steps with Authorize and capture transactions.

-------------------


**Post merge**
